app/src/test/java/org/mozilla/tv/firefox/ext/LiveDataPostIfNewTest.kt

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.ext
+
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.Observer
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.spy
+import org.mockito.Mockito.times
+import org.mockito.Mockito.verify
+import org.mozilla.tv.firefox.utils.PreventLiveDataMainLooperCrashRule
+
+@Suppress("TestFunctionName")
+class LiveDataPostIfNewTest {
```

Comment:
nit: unit tests are usually named after the class or file name: why is this named after a function?

That being said, in general more encapsulation for unit tests seems reasonable (though it duplicates the fact that the method is called in the test function anyway) but I wonder if this is the way to do it?

Reply:
Having the helper methods live in one file seems easiest from a discoverability perspective, but each one needs a pretty substantial amount of testing.  Those tests also don't really share a lot of code, so keeping them together would only serve to make them more difficult to read.  This seemed appropriate at the time, but I'm open to alternate suggestions as well.